Header header
float64[] q
float64[] q_dot
float64[] q_des
float64[] q_dot_des
float64[] M
float64[] C
float64[] G
float64[] J
float64[] J_dot
float64[] J_aux
float64[] M_aux
float64[] torque_cmd
float64[] p
float64[] p_des
float64[] P_dot
float64[] P_dot_des
float64[] P_ddot_cmd

